src/log.js

```diff
@@ -4,17 +4,20 @@ import {
   createLogger,
   createRoarrInititialGlobalState,
 } from './factories';
+import globalThis from './getGlobalThis';
 
-global.ROARR = createRoarrInititialGlobalState(global.ROARR || {});
+if (globalThis === global) {
+  globalThis.ROARR = createRoarrInititialGlobalState(globalThis.ROARR || {});
```

Comment:
Just to be explicit, I would rather use `detect-node` here than check for `global` presence.

https://github.com/iliakan/detect-node